### PR TITLE
Backend fixes and integration

### PR DESCRIPTION
**Pull request by FilipParyz**
Some fixes and integration with frontend to display things properly

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -93,11 +98,11 @@
...
                     <div id="info-box-Right">
                         <div class="label" id="label-weight">Weight</div>
-                            <label class="paragraph" id="paragraphWeight">0</label><div></div>
+                            <label class="paragraph" id="paragraphWeight">{{ animal.weight }}</label><div></div>
                         <div class="label" id="label-status">Status</div>
-                            <label class="paragraph" id="paragraphStatus">Adopted</label><div></div>
+                            <label class="paragraph" id="paragraphStatus">{{ animal.status }}</label><div></div>
                         <div class="label" id="label-sex">Sex</div>
-                            <label class="paragraph" id="paragraphlSex">Female</label><div></div>                        
+                            <label class="paragraph" id="paragraphlSex">{{ animal.status }}</label><div></div>
```
Comment (Mariusz-Borczuk) [inline templates/index.html:105]: Status to płeć?

Comment (FilipParyz) [reply templates/index.html]: Nope, that was a mistake, fixed that in the newest commit.

**Review by Mariusz-Borczuk — CHANGES_REQUESTED** [review]
Did you checked whether {{animall.ATTRIBUTE}} applies to every applies to every single instation of profile? as it doesnt show when I turn it on

  Also in line 105 index.html  <label class="paragraph" id="paragraphlSex">{{ animal.status }}</label><div></div> animal.status should be animal.sex right?

**Review by FilipParyz — COMMENTED** [review]
Everything should work as intended.

**Review by Mariusz-Borczuk — APPROVED** [review]

**Pull request merged**